components/affix/style/index.less

```diff
@@ -1,4 +1,4 @@
-@import "../../style/themes/default";
+@import '../../style/themes/default';
```

Comment:
Glad to see that `ant-design` team has adopted prettier to maintain a consistent code style.